mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxACL.java

```diff
@@ -383,6 +383,11 @@ public static EntryKey createUserEntryKey(Username name, boolean negative) {
             return new EntryKey(name.asString(), NameType.user, negative);
         }
 
+        public static EntryKey createUserOrSpecialEntryKey(Username name)  {
```

Comment:
`or` in a method name?  What to expect?

Sorry, reflex. Naming should be improved.